### PR TITLE
⚠️ Safety First ⚠️

### PR DESCRIPTION
**Pull request by nolim1t**
Added dependency checking to bash scripts

### CODE REVIEW

File: configure-box.sh [hunk 1]
```diff
@@ -11,6 +11,17 @@
...
+check_dependencies () {
+  for cmd in "$@"; do
+    if ! command -v $cmd >/dev/null 2>&1; then
+      echo "This script requires \"${cmd}\" to be installed"
+      exit 1
+    fi
+  done
+}
+
+check_dependencies wget docker
```
Comment (mayankchhabra) [inline configure-box.sh:23]: Can you also check for `docker-compose` here?

Comment (nolim1t) [reply configure-box.sh:23]: Actually docker-compose isn't required to run the install. Everything can be set up without it.

The main purpose of this is to make it easy to install and configure.

**Comment by nolim1t** [comment]
FYI.. the checks are only whats needed to complete the task at hand.  That is installing  and configuring.

**Comment by mayankchhabra** [comment]
Yep, `docker-compose pull` is run inside `configure-box.sh`
https://github.com/getumbrel/umbrel/blob/b47d1cb16d9e62bd8c48977db13870a65b6f38af/configure-box.sh#L80

**Comment by nolim1t** [comment]
Added while resolving the conflict

**Pull request merged**